extensions/vault/runtime/src/main/java/io/quarkus/vault/internal/VaultAdManager.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.vault.internal;
```

Comment:
Everything internal should be under the `runtime` subpackage.

Reply:
done https://github.com/quarkusio/quarkus/pull/2897#discussion_r326828197